index.html

```diff
@@ -13,21 +13,39 @@
     />
   </head>
   <body>
+    <header class="head">
+      <div class="logo">My logo</div>
+      <nav id="navbar1"  onclick="showNav">
+        <img src="image/Union.png" alt="button" />
+      </nav>
+      <nav class="navbar">
+        <ul class="container">
+          <li><a href="#facebook">Portfolio</a></li>
+          <li><a href="#About">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <!-- ----------------- hamburger ------------------- -->
+    <div class="ham" id="burger">
+      <button id="menuIcon" class="close-icon" onclick="closeNavbar()">
+        <img src="image/cancel.png" alt="" />
+      </button>
+      <ul class="hambar">
+        <li onclick="hideNavbar()">
+          <a href="#facebook">Portfolio</a>
+        </li>
+        <li onclick="hideNavbar()">
+          <a href="#About">About</a>
+        </li>
+        <li onclick="hideNavbar()">
+          <a href="#contact">Contact</a>
+        </li>
```

Comment:
- [ ] Nice work on hidding the navbar when the user clicks on any of the menu links 👏 however fo javascript best practice all events should be in the js file. thus I think it would be better to to use the onclick events in the js file rather that in the html file.